pkgs/development/interpreters/erlang/generic-builder.nix

```diff
@@ -103,6 +105,31 @@ in stdenv.mkDerivation ({
 
   setupHook = ./setup-hook.sh;
 
+  passthru = {
+    updateScript =
+      let major = builtins.head (builtins.splitVersion version);
+      in writeScript "update.sh" ''
+      #!${stdenv.shell}
+      set -ox errexit
+      PATH=${
+        lib.makeBinPath [
+          common-updater-scripts
+          coreutils
+          git
+          gnused
+        ]
```

Comment:
```suggestion
        lib.makeBinPath [ common-updater-scripts coreutils git gnused ]
```

Reply:
I'm fairly strongly in favor of multiple lines here as it makes future diffs easier to reason about. Evan, the author of Elm, had a good blog post about this a while back, but I can't seem to find it quickly. Can you help me understand your reasoning behind this suggestion?

Reply:
I wanted to shorten the amount of lines to make the derivation not bigger as it is. If you like it this way out is fine for me, too and it won't block the merge.